### PR TITLE
[proto-definitions - Part 1] Deleting Unused HorizontalRule

### PR DESCRIPTION
**Pull request by Marcial1234**
Part of #247

### CODE REVIEW

File: third_party/tutorial.proto [hunk 1]
```diff
@@ -45,15 +45,10 @@ message BlockContent {
     Image image = 2;
     InfoBox info_box = 3;
     CodeBlock code_block = 4;
-    HorizontalRule horizontal_rule = 5;
```
Comment (cassierecher) [inline third_party/tutorial.proto]: [Reserve](https://developers.google.com/protocol-buffers/docs/proto#reserved) these.

Comment (Marcial1234) [reply third_party/tutorial.proto]: Done

File: third_party/tutorial.proto [hunk 2]
```diff
@@ -46,6 +46,8 @@ message BlockContent {
     InfoBox info_box = 3;
     CodeBlock code_block = 4;
     Table table = 6;
+
+    reserved 5;
```
Comment (cassierecher) [inline third_party/tutorial.proto]: Also reserve the name "horizontal_rule"

Comment (Marcial1234) [reply third_party/tutorial.proto]: Done

File: third_party/tutorial.proto [hunk 3]
```diff
@@ -45,13 +45,10 @@ message BlockContent {
...
     InfoBox info_box = 3;
     CodeBlock code_block = 4;
-    HorizontalRule horizontal_rule = 5;
     Table table = 6;
-  }
-};
 
-message HorizontalRule {
-  // No fields.
+    reserved 5, "horizontal_rule";
```
Comment (cassierecher) [inline third_party/tutorial.proto]: I think it's not possible to combine numbers and names. Try:

reserved 5;
reserved "horizontal_rule";

Comment (Marcial1234) [reply third_party/tutorial.proto]: You are correct.

**Pull request merged**